src/App.js

```diff
@@ -52,6 +53,7 @@ const App = () => {
         <Route path="/upload" component={UploadForm} />
         <Route path="/upload-save" component={UploadSudoku2} />
         <Route path="/about" component={TeamList} />
+        <Route path="/soon" component={SoonPage} />
```

Comment:
i'm assuming you just went ahead and added the route while you were working on the instruction page and that is why there is more than one PR on here?

Reply:
Looking at this PR and the other [PR](https://github.com/Lambda-School-Labs/omega2020-fe/pull/88/commits/6c8df3432920120392a618f081dc9f58430bd20a)
 changes, they have they same file changes other then the 2 extra files for instructions

Reply:
so do you want him to create a whole new pr on the other branch then?

Reply:
ok merging the other PR to master, then pulling changes from master resolved the issue I was seeing. Good to merge